homeassistant/components/shelly/icons.json

```diff
@@ -0,0 +1,45 @@
+{
+  "entity": {
+    "button": {
+      "mute": {
+        "default": "mdi:volume-mute"
+      },
+      "self_test": {
+        "default": "mdi:progress-wrench"
+      },
+      "unmute": {
+        "default": "mdi:volume-high"
+      }
+    },
+    "number": {
+      "valve_position": {
+        "default": "mdi:pipe-valve"
+      }
+    },
+    "sensor": {
+      "gas_concentration": {
+        "default": "mdi:gauge"
+      },
+      "lamp_life": {
+        "default": "mdi:progress-wrench"
+      },
+      "operation": {
+        "default": "mdi:cog-transfer"
+      },
+      "tilt": {
+        "default": "mdi:angle-acute"
+      },
+      "valve_status": {
+        "default": "mdi:valve"
+      }
+    },
+    "switch": {
+      "valve_switch": {
+        "default": "mdi:valve-open",
+        "state": {
+          "off": "mdi:valve-closed"
```

Comment:
Before it use to be open only if on, and defaults to close, now it's the other way around.

Or event better:
A "basic" valve icon as default and specify open on "on" state an close on "off" state

Reply:
Good idea, thanks.